ramls/schemas/fieldName.json

```diff
@@ -0,0 +1,39 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Field names",
+  "type": "object",
+  "additionalProperties": false,
+    "properties": {
+      "id": {
```

Comment:
This is not a UUId according to the example above. Also let's not use id as the field name if its not UUID to avoid confusions like this

Reply:
I will rename it to the ruleId